src/main/groovy/com/netflix/nebula/lint/rule/dependency/DependencyHelper.groovy

```diff
@@ -0,0 +1,93 @@
+package com.netflix.nebula.lint.rule.dependency
+
+import com.netflix.nebula.lint.GradleViolation
+import com.netflix.nebula.lint.rule.GradleDependency
+import org.codehaus.groovy.ast.expr.ClosureExpression
+import org.codehaus.groovy.ast.expr.ConstantExpression
+import org.codehaus.groovy.ast.expr.GStringExpression
+import org.codehaus.groovy.ast.expr.MapExpression
+import org.codehaus.groovy.ast.expr.MethodCallExpression
+import org.codehaus.groovy.ast.expr.NamedArgumentListExpression
+
+class DependencyHelper {
+    static void removeVersion(GradleViolation violation, MethodCallExpression call, GradleDependency dep) {
+        if (call.arguments.expressions.size == 1 && call.arguments.expressions[0] instanceof ConstantExpression) {
+            handleConstantExpression(violation, call.arguments.expressions[0], "'${dep.group}:${dep.name}'")
+        } else if (call.arguments.expressions.size == 1 && call.arguments.expressions[0] instanceof GStringExpression) {
+            handleGStringExpression(violation, call.arguments.expressions[0], "'${dep.group}:${dep.name}'")
```

Comment:
This could still have interpolated variables right? You'd want to preserve the `"`?